config/network.go

```diff
@@ -2,24 +2,48 @@ package config
 
 import (
 	"encoding/base64"
+	"errors"
 	"fmt"
 	"os"
 	"strings"
 
-	"errors"
-
 	"github.com/pelletier/go-toml/v2"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
 )
 
 const (
 	Base64NetworkConfigEnvVarName = "BASE64_NETWORK_CONFIG"
 )
 
+type ForkConfig struct {
+	URL              string `toml:"url"`          // URL is the URL of the node to fork from
+	BlockNumber      int64  `toml:"block_number"` // BlockNumber is the block number to fork from
+	BlockTime        int64  `toml:"block_time"`
+	Retries          int    `toml:"retries,omitempty"`
+	Timeout          int64  `toml:"timeout,omitempty"`
+	ComputePerSecond int64  `toml:"compute_per_second,omitempty"`
+	RateLimitEnabled bool   `toml:"rate_limit_enabled,omitempty"`
```

Comment:
nit pick: add comments for these two values? not sure what `block_time` or `compute_per_second` means.